FileGetContentsLoader.php

```diff
@@ -28,7 +28,7 @@ class FileGetContentsLoader
      *
      * @throws JsonLdException If loading the document failed.
      */
```

Comment:
Could you please replace the DocBlock with

    /**
     * {@inheritdoc}
     */